destination/src/main/scala/quasar/plugin/sqlserver/destination/TempTableFlow.scala

```diff
@@ -88,14 +90,22 @@ object TempTableFlow {
     } yield {
       val flow = new TempTableFlow {
         def ingest(chunk: Chunk[CharSequence]): ConnectionIO[Unit] =
-          tempTable.ingest(chunk) >> commit
+          retry(tempTable.ingest(chunk)) >>
+          retry(commit)
```

Comment:
If we `rollback` on error prior to retry, instead of retrying each action I think we could retry their composition instead.

Reply:
:+1: